src/assignment/views/AssignmentOverview.tsx

```diff
@@ -1,3 +1,5 @@
+import './AssignmentOverview.scss';
```

Comment:
We should get the eslint import ordering to work for scss files, so scss files are automatically sorted to the top

Reply:
With a quick search it doesn't look like it's out-of-the-box due to some import order nuance